src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -115,7 +189,17 @@ static void create_data(ProgramBuilder& p, const ov::Shape& const_shape, const s
         auto buf = lock.data();
         auto bufSize = constLayout.bytes_count();
 
-        std::memcpy(&buf[0], &data[0], bufSize);
+        if (constLayout.data_padding) {
```

Comment:
isn't it necessary to check also `data_type == u4 || data_type == i4`?

Reply:
data_padding is only filled under the following condition
`p.get_config().get_use_onednn()
            && (op->get_element_type() == ov::element::i4 || op->get_element_type() == ov::element::u4)
            && !op->is_dynamic()
            && const_shape.size() == 2`, So I think if data_padding is not empty, it is i4 or u4 data types. Are there any other cases? thanks.